glean-core/tests/timing_distribution.rs

```diff
@@ -314,3 +314,33 @@ fn large_nanoseconds_values() {
     // Check that we got the right sum and number of samples.
     assert_eq!(val.sum() as u64, time);
 }
+
+#[test]
+fn stopping_non_existing_id_records_an_error() {
+    let (glean, _t) = new_glean(None);
+
+    let mut metric = TimingDistributionMetric::new(
+        CommonMetricData {
+            name: "non_existing_id".into(),
```

Comment:
I don't understand why the naming choice of `non_existing_id` here.

Reply:
Because when you stop a timing distribution, it looks for the id of the timer passed in. For this metric we're using a non existing id to trigger an error recording :)